src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -858,16 +859,20 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, description,
 	);
 
 	const renderBreadcrumbs = (): ReactNode => {
-		const defaultBreadcrumbs: Breadcrumb[] = [
-			{
-				label: `${tText('pages/slug/ie/index___breadcrumbs___home')}`,
-				to: ROUTES.home,
-			},
+		const defaultBreadcrumbs: Breadcrumb[] = flatten([
+			...(!isKiosk
+				? [
+						{
+							label: `${tText('pages/slug/ie/index___breadcrumbs___home')}`,
+							to: ROUTES.home,
+						},
+				  ]
+				: []),
```

Comment:
no need to flatten a flat array
if statements with negative conditions are harder to read
```suggestion
		const defaultBreadcrumbs: Breadcrumb[] = [
			...(isKiosk
				? []
				: [
						{
							label: `${tText('pages/slug/ie/index___breadcrumbs___home')}`,
							to: ROUTES.home,
						},
				  ],
```